README.md

```diff
@@ -29,6 +29,7 @@ public io.opnetracing.Tracer tracer() {
 ## Development
 ```shell
 ./mvnw clean install
+make -C opentracing-spring-cloud-starter // to run dependency tests, a specific profile can be specified by PROFILES=nodeps
```

Comment:
I like how you think, but I'd avoid using `make` here, for this purpose. It's not that common for Java developers to have `make` available, specially for developers on Windows.

Do you need to run `mvn clean test` for each profile, or can you run both at the same time, like `mvn clean test -Pnodeps,feign`? In the worst case, just add the two commands to both Travis and the readme. *If* the number of steps grows dramatically, then it might be worth requiring `make` and automating it further there (or even writing a simple maven plugin, if none exists already).

Reply:
`mvn clean test -Pnodeps,feign` does not do what I want. 

I want to run profiles in sequence with requires some sort of loop.

Reply:
If you add each command to one stage, you could run them in parallel in Travis and don't need a `Makefile`. Bonus points for not requiring future contributors to worry about having or not `make` in the path. Just make sure to add enough documentation to the readme.

https://docs.travis-ci.com/user/build-stages/

Reply:
agree about travis config, however I would like to have also one liner wich can be used to spin all tests locally. e.g. in jaeger it's quite complicated to run all tests/builds locally to verify that travis build will eventually pass.